.ncurc.yml

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
Please add the following

```yaml
## Add a TODO comment indicating the reason for each rejected dependency upgrade added to this list, and what should be done to resolve it (i.e. handle it through a story, etc).
reject: [
  // @hapi/catbox-memory v6 and onwards breaks tests. Needs investigation.
  "@hapi/catbox-memory"
  # TODO: v6+ (ref: https://github.com/sindresorhus/get-port/releases/tag/v6.0.0) is an ESM library and thus not compatible with CommonJS. Future story needed to resolve.
  "get-port"
]
```

And  remove `.ncurc.js`

Reply:
I thought we were using `.js` as the standard filetype, am I mistaken? What filetype are we setting on?

Reply:
Seems I'm mistaken :thinking: wonder where I got js into my head.